packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -406,6 +407,7 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
       enableImpeller: ImpellerStatus.fromBool(argResults!['enable-impeller'] as bool?),
       debugLogsDirectoryPath: debugLogsDirectoryPath,
       webRenderer: webRenderer,
+      printDtd: boolArg(FlutterGlobalOptions.kPrintDtd, global: true),
```

Comment:
we seem to repeat a lot of this arg parsing logic throughout flutter tools. Could be nice if there were some common getters to use

```
bool get printDtdFlag => boolArg(FlutterGlobalOptions.kPrintDtd, global: true);
bool get enableDevToolsFlag => boolArg(FlutterCommand.kEnableDevTools);
bool get disableServiceAuthCodesFlag => boolArg('disable-service-auth-codes');
```

but likely out of scope for this PR as this would be a larger cleanup